one.lfa.android.services/src/main/res/values-so/strings.xml

```diff
@@ -125,7 +125,7 @@
 <string name="profileAskGender">jinsigeed tahay?</string>
 <string name="profileAskGrade">Fasalkee tahay?</string>
 <string name="profileAskName">magacaa?</string>
-<string name="profileAskPilotSchool">dugsigaagu ma waxa weeye barnaamijka eLibrary paylood</string>
+<string name="profileAskPilotSchool">dugsigaagu ma waxa weeye barnaamijka Library For All</string>
```

Comment:
We need to leave the `paylood` in this translation.

Reply:
Solved in the latest commit.